docs/bitcoin/quicksync.md

```diff
@@ -0,0 +1,27 @@
+# QuickSync
+
+## Overview
+QuickSync downloads a torrent for a major chunk of the blockchain from MyNode peers without verifying every block.
+It saves time by avoiding the initial blockchain download and this time benefit is significant when using an HDD but marginal with a SSD. QuickSync is turned on by default and must be disabled from the settings page while setting up to trigger fully-verified blockchain download.
+
+## Tradeoffs
+
+- **Trust vs Time**: QuickSync downloads a torrent of the Bitcoin blockchain synced by the MyNode creator. This chunk of the blockchain saves the time in downloading verifying the blockchain from scratch. This convenience has a cost. The user is trusting that MyNode is propagating a legitimate copy of the Bitcoin blockchain. To minimize trust, one should disable QuickSync and download the blockchain from scratch. In case you have already synced the blockchain using QuickSync, you can [reset the blockchain](/bitcoin/troubleshoot.html#reset-blockchain).
+
+- **Memory vs Community support**: QuickSync downloads a compressed form of the blockchain. This compressed file is not deleted after its expanded form is used for boosting the Bitcoin sync process. Instead, this file is seeds torrent for the other MyNode users to improve their download speed. The compressed file uses around 236 GB. To save this space, one should disable QuickSync.
+
+<!-- attach a table of expected times -->
+
+## Enable/Disable
+
+Once the blockchain is synced with the network, enabling QuickSync won't affect the Bitcoin blockchain. Instead, it will just download the torrent and start seeding it. This helps MyNode peers in downloading the blockchain faster during setup. The upload and download speed can be adjusted to optimize bandwidth usage, as shown in the screenshot below.
```

Comment:
```suggestion
Once the blockchain is synced with the network, enabling QuickSync won't affect the Bitcoin blockchain. Instead, it will just download the torrent and start seeding it. This helps myNode peers in downloading the blockchain faster during setup. The upload and download speed can be adjusted to optimize bandwidth usage, as shown in the screenshot below.
```